docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -307,6 +307,12 @@ With the aforementioned example in your `application.properties`, your Hibernate
 data both during the JVM mode test run and during the native mode test run. The production
 executable will contain only the `version.txt` resource, no superfluous test data.
 
+[NOTE]
+====
+Unlike the JVM mode, in native it's not possible to specify build-time properties in `application.properties` inside the `src/test` directory.
+Only use `src/main` for this.
+====
+
```

Comment:
This is not only for the native image. The same applies if we run Quarkus jar in integration-tests (like we do with the native image).

Reply:
Oh, right, I misunderstood. How about now? I moved it to `getting-started-testing.adoc`